src/NHibernate/Dialect/Dialect.cs

```diff
@@ -2063,6 +2074,15 @@ public virtual bool SupportsSubSelects
 		/// </summary>
 		public virtual bool SupportsPoolingParameter => true;
 
+		/// <summary>
+		/// Does this dialect support distributed transaction?
+		/// </summary>
+		/// <remarks>
+		/// Distributed transactions usually imply the use of<see cref="TransactionScope"/>, but using
+		/// <c>TransactionScope</c> does not imply the transaction will be distributed.
+		/// </remarks>
+		public virtual bool SupportsDistributedTransactions => true;
```

Comment:
It looks in most case more the data-provider "fault" when not supported than the database itself. Maybe should it be moved to driver.